### PR TITLE
Update Module.js

### PR DESCRIPTION
**Pull request by tudoranghelina91**
Added var keyword on lines 41. 45 and 49 as requested in https://github.com/publiclab/image-sequencer/issues/1084

Fixes #0000  (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by aashna27** [comment]
@tudoranghelina91 this is all really great, but I am afraid someone else had claimed it before.

**Comment by tudoranghelina91** [comment]
> @tudoranghelina91 this is all really great, but I am afraid someone else had claimed it before.

Thank you for your reply

File: src/modules/ColorTemperature/Module.js [hunk 1]
```diff
@@ -38,15 +38,15 @@ module.exports = function ColorTemperature(options, UI) {
       for (let i = 0; i < pixels.shape[0]; i++) {
         for (let j = 0; j < pixels.shape[1]; j++) {
 
-          r_data = pixels.get(i, j, 0);
+          var r_data = pixels.get(i, j, 0);
```
Comment (aashna27) [inline src/modules/ColorTemperature/Module.js]: the is a comma at the end so that the var can be shared with the other variable

Comment (tudoranghelina91) [reply src/modules/ColorTemperature/Module.js]: Hello, @aashna27 , could you please elaborate?

**Comment by HarshKhandeparkar** [comment]
Not required anymore

**Pull request closed without merging**